hearts/api/socket_events.py

```diff
@@ -66,3 +82,43 @@ def on_leave(data):
     room = data['room']
     io.leave_room(room)
     chat(username + ' has left the room.', room=room)
+
+
+@socketio.on('pass_cards')
+def on_pass_cards(data):
+    '''
+    data: {'cards': [str, str, str]}
+    '''
+    socket_id = request.sid
+
+    '''
+    delete this later
+    '''
+    if 'game' in session:
+        print('pass cards: game is in session')
+        game_id = session['game']
+    else:
+        print('pass cards: game is not in session')
+        raise Exception
```

Comment:
Look up the game id here based on the room id, and store it on the session for later use.